CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## 2.19.0-rc.0 / 2020-06-08
+
+* [FEATURE] Discovery: Added discovery support for Triton global zones. #7250
+* [FEATURE] TSDB: It now accepts callbacks to control series creation if you use TSDB as a package. #7159 
+* [FEATURE] Added time range parameters for label names and label values API. #7288
+* [ENHANCEMENT] TODO. #7188
+* [ENHANCEMENT] TSDB: Memory-map full chunks of Head (in-memory) block from disk. This reduces memory footprint and makes restarts faster. #6679
+* [ENHANCEMENT] Increased timeout for alertmanager send failures. #7228
```

Comment:
This is misleading, it sounds like it's the http request timeout we changed. This is about resend-relay being more tolerant of failures.